CMakeLists.txt

```diff
@@ -135,7 +135,10 @@ option(libswiftnav_ENABLE_TESTS "" OFF)
 find_package(Swiftnav REQUIRED)
 
 add_subdirectory(src)
-add_subdirectory(python)
+
+if(NOT LIBSETTINGS_SKIP_PYTHON)
```

Comment:
`libsettings_ENABLE_PYTHON` was introduced here: https://github.com/swift-nav/libsettings/pull/40 and just got merged into master